api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -586,4 +586,30 @@ public Locale getLocale() {
 	public void setLocale(Locale locale) {
 		this.locale = locale;
 	}
+
+	/**
```

Comment:
It would be great to add the method description.

Reply:
Sure, but that is something that was create in 2013, I added a little description [here](https://github.com/openmrs/openmrs-module-uiframework/pull/65/files#diff-6e177127ee5b17a0d0a088acefa4c4d980d03a506959223081b579f153e8382fR587-R591)

Reply:
oh ok.